### PR TITLE
Bump lerna to 5.x

### PR DESCRIPTION
**Pull request by fcollonval**
## References

Bump lerna to 5.x

## Code changes

N/A

## User-facing changes

N/A

## Backwards-incompatible changes

None

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -119,7 +119,7 @@
     "stylelint-prettier": "^2.0.0"
   },
   "devDependencies": {
-    "lerna": "^4.0.0",
+    "lerna": "^5.0.0",
```
Comment (jtpio) [inline package.json:122]: Maybe we could set the version to `^5.4.2` here for consistency with the new version resolved in `yarn.lock`?

```suggestion
    "lerna": "^5.4.2",
```

Comment (jtpio) [reply package.json:122]: Actually a `5.4.3` was released a couple of hours ago and we could also update to it: https://github.com/lerna/lerna/releases/tag/v5.4.3

**Review by jtpio — APPROVED** [review]
Thanks!

**Comment by fcollonval** [comment]
I opened #12957 to fix the regression on the documentation snapshot

**Comment by jtpio** [comment]
> I opened #12957 to fix the regression on the documentation snapshot

Thanks :+1:. This is indeed not related to the change in this current PR.

**Pull request merged**